### PR TITLE
doc: fix all docs and make output for doc

### PR DESCRIPTION
**Pull request by yihong0618**

### CODE REVIEW

File: greenplumpython/type.py [hunk 1]
```diff
@@ -12,12 +12,28 @@ class TypeCast(Expr):
...
-                regclass = gp.get_type("regclass", db)
-                dataframe_name = regclass(series["tableoid"]).rename("dataframe_name")
+                >>> rows = [(i,) for i in range(10)]
+                >>> series = db.create_dataframe(rows=rows, column_names=["val"]).save_as("series", column_names=["val"])
+                >>> regclass = gp.get_type("regclass")
+                >>> dataframe_name = series.assign(dataframe_name=lambda t: regclass(t["tableoid"]))
+                ----------------------
+                val | dataframe_name
+                -----+----------------
+                0 | serie
```
[the remark this answers is not in the document]
Comment (yihong0618) [reply greenplumpython/type.py:25]: nop I copy and paste... this is my wrong

[the remark this answers is not in the document]
Comment (yihong0618) [reply greenplumpython/type.py]: yes....

File: greenplumpython/dataframe.py [hunk 2]
```diff
@@ -303,24 +303,58 @@ def apply(
...
+            .. highlight:: python
             .. code-block::  python
 
-                @gp.create_function
-                def label(prefix: str, id: int) -> str:
-                    prefix = "id"
-                    return f"{prefix}_{id}"
-
-                result = series.apply(lambda t: label("label", t["id"]))
+                >>> @gp.create_function
```
Comment (beeender) [inline greenplumpython/dataframe.py:336]:
```
>>> rows = [(i,) for i in range(-10, 0)]
>>> series = db.create_dataframe(rows, column_names=["id"])
```

Add those, minus ID doesn't look good

**Review by beeender — COMMENTED** [review]
LOOKS GREAT!!

File: greenplumpython/dataframe.py (hunk 2 above)
Comment (yihong0618) [reply greenplumpython/dataframe.py:336]: copy that

**Review by beeender — APPROVED** [review]

File: greenplumpython/dataframe.py (hunk 2 above)
Comment (yihong0618) [reply greenplumpython/dataframe.py:338]: fixed

**Pull request merged**